freight/notifiers/webhook.py

```diff
@@ -0,0 +1,52 @@
+__all__ = ["WebhookNotifier"]
+
+from freight import http
+from freight.models import App, Task, TaskStatus, User
+
+from .base import Notifier, NotifierEvent, generate_event_title
+
+
+def stringify_date(date):
+    return date.isoformat() + "Z" if date else None
+
+
+class WebhookNotifier(Notifier):
+    def get_options(self):
+        return {"webhook_url": {"required": True}, "headers": {"required": False}}
```

Comment:
Maybe just call this `url` instead of `webhook_url` now since the entire plugin is called `WebhookNotifier` it's redundant imo.